docs/hugo/content/introduction/authentication.md

```diff
@@ -36,6 +43,98 @@ stringData:
 EOF
 ```
 
+## Azure Workload Identity
+
+### Prerequisites
+1. An existing Azure Service Principal or Managed Identity. The setup is the same regardless of which you choose.
+2. The [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli).
+3. The [Azure Workload Identity](https://github.com/Azure/azure-workload-identity) webhook installed into your cluster. See [Azure Workload Identity installation](https://azure.github.io/azure-workload-identity/docs/installation.html). Note that setup involves two steps, getting your clusters OIDC issuer URL and installing the [Azure Workload Identity mutating webhook](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html).
+
+Use the following Bash script to set the enviornment variables required for the below commands:
+```bash
+export AZURE_CLIENT_ID="00000000-0000-0000-0000-00000000000"       # The client ID (sometimes called App Id) of the Service Principal, or the Client ID of the Managed Identity which you are using Workload Identity with.
+export AZURE_SUBSCRIPTION_ID="00000000-0000-0000-0000-00000000000" # The Azure Subscription ID the identity is in.
+export AZURE_TENANT_ID="00000000-0000-0000-0000-00000000000"       # The Azure AAD Tenant the identity/subscription is associated with.
+export SERVICE_ACCOUNT_ISSUER="https://oidc.prod-aks.azure.com/00000000-0000-0000-0000-00000000000/" # The OIDC endpoint for your cluster in this example AKS
+```
+
+### Get the Application Object ID of the identity
+For Service Principal: `export APPLICATION_OBJECT_ID="$(az ad app show --id ${AZURE_CLIENT_ID} --query id -otsv)"`
+   
+For Managed Identity: `export APPLICATION_OBJECT_ID=$(az resource show --id /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/{my resource group name}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{my managed identity name} --query "properties.principalId" -o tsv | tr -d '[:space:]')`
```

Comment:
Suggest moving these code blocks into explicit code blocks rather than inline. This will give them more room to stretch horizontally. You may also want to wrap the second one since it's quite long.

Reply:
I've rewritten this section a bit, given that actually MI and SP are more different than I originally realized. As part of that. I rolled these commands into the existing code blocks.